### PR TITLE
Specify how to navigate to option

### PR DESCRIPTION
**Pull request by get-itips**
closes https://github.com/MicrosoftDocs/OfficeDocs-SharePoint/issues/3445

### CODE REVIEW

File: SharePoint/SharePointOnline/manage-sites-in-new-admin-center.md [hunk 1]
```diff
@@ -47,20 +47,24 @@ For more info about tasks on the Active sites page, see:
 
 For all site types except channel sites, you can add or remove site admins and change the primary admin. For group-connected team sites, you can also add and remove group owners. Note that if you remove a person as a primary admin, they will still be listed as an additional admin. For info about each role, see [About site permissions](site-permissions.md).
 
-1. In the left column, select a site. 
+1. In the SharePoint admin center, select Sites > Active sites or browse to <a href="https://go.microsoft.com/fwlink/?linkid=2185220" target="_blank">**Active sites** page</a>.
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointOnline/manage-sites-in-new-admin-center.md:50]:
```suggestion
1. In the SharePoint admin center, select **Sites** > **Active sites** or browse to the <a href="https://go.microsoft.com/fwlink/?linkid=2185220" target="_blank">**Active sites** page</a>.
```

File: SharePoint/SharePointOnline/manage-sites-in-new-admin-center.md [hunk 2]
```diff
@@ -47,20 +47,24 @@ For more info about tasks on the Active sites page, see:
...
+
+2. In the left column, select a site. 
     
-2. Select **Permissions**. 
+3. Select **Permissions**. 
 
-3. Click **Manage** for the permissions that you want to update.
+4. Click **Manage** for the permissions that you want to update.
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointOnline/manage-sites-in-new-admin-center.md:56]:
```suggestion
4. Select **Manage** for the permissions that you want to update.
```

File: SharePoint/SharePointOnline/manage-sites-in-new-admin-center.md [hunk 3]
```diff
@@ -47,20 +47,24 @@ For more info about tasks on the Active sites page, see:
...
 
-4. Add or remove people or change their role, and then click **Save**.
+5. Add or remove people or change their role, and then click **Save**.
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointOnline/manage-sites-in-new-admin-center.md:58]:
```suggestion
5. Add or remove people or change their role, and then select **Save**.
```

File: SharePoint/SharePointOnline/manage-sites-in-new-admin-center.md [hunk 4]
```diff
@@ -47,20 +47,24 @@ For more info about tasks on the Active sites page, see:
...
     
 ## Change a site's hub association
 <a name="hubsite"> </a>
   
-1. In the left column, select a site. 
+1. In the SharePoint admin center, select Sites > Active sites or browse to <a href="https://go.microsoft.com/fwlink/?linkid=2185220" target="_blank">**Active sites** page</a>.
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointOnline/manage-sites-in-new-admin-center.md:63]:
```suggestion
1. In the SharePoint admin center, select **Sites** > **Active sites** or browse to the <a href="https://go.microsoft.com/fwlink/?linkid=2185220" target="_blank">**Active sites** page</a>.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@MikePlumleyMSFT PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**